app/services/user.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcrypt');
+const { Sequelize, ...db } = require('../models');
+const errors = require('../errors');
+
+const { Op } = Sequelize;
+
+const toEncrypt = data => {
+  const saltRounds = 5;
+  const salt = bcrypt.genSaltSync(saltRounds);
+  const hash = bcrypt.hashSync(data, salt);
+  return hash;
+};
```

Comment:
This encryption logic should not go in the services, it should go in the helpers.